app/assets/javascripts/discourse/app/templates/composer.hbs

```diff
@@ -89,12 +89,14 @@
 
         <div class="submit-panel">
           <PluginOutlet @name="composer-fields-below" @tagName="span" @connectorTagName="div" @args={{hash model=this.model}} />
-
           <div class="save-or-cancel">
             {{#unless this.model.viewFullscreen}}
               <ComposerSaveButton @action={{action "save"}} @icon={{this.saveIcon}} @label={{this.saveLabel}} @forwardEvent={{true}} @disableSubmit={{this.disableSubmit}} />
 
               {{#if this.site.mobileView}}
+                {{#if this.model.canSaveDraft}}
+                  <DButton @class="btn-flat save-draft" @action={{action "saveDraft"}} @icon="far-save"/>
```

Comment:
Maybe a more specific class name for this if we are keeping the modal one too could be `composer-save-draft`